docs/source/user_guide/troubleshooting.rst

```diff
@@ -44,3 +44,13 @@ Thus, the following cases will fail:
 - partially
 
   - pre-compilation of ``noarch`` packages, with no impact on capability to use the package but removing it will let artifacts (pycache) on the filesystem
+
+
+Hangs during install in QEMU
+----------------------------
+When using Mamba/Micromamba inside a QEMU guest, installing packages may sometimes hang forever due to an `issue with QEMU and glibc <https://gitlab.com/qemu-project/qemu/-/issues/285>`_. As a workaround, set ``G_SLICE=always-malloc`` in the QEMU guest, eg.::
+
+  export G_SLICE=always-malloc
```

Comment:
Is there still any point to suggesting `micromamba config set extract_threads 1`? (If so, we should offer the corresponding `mamba` command...) Or is that now pointless after your patch?

Reply:
I think the `G_SLICE` solution is much better because it doesn't slow down the process noticably.

Reply:
G_SLICE doesn't seem to work for me. However doing `RUN micromamba config set extract_threads 1` before `RUN micromamba create -vv -y -n pangolin -f /environment.yml` _does_ work luckily!